packages/edit-site/src/components/header/navigation-link/index.js

```diff
@@ -11,22 +11,33 @@ import { __ } from '@wordpress/i18n';
 import { wordpress } from '@wordpress/icons';
 import { store as coreDataStore } from '@wordpress/core-data';
 import { useReducedMotion } from '@wordpress/compose';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../../store';
 
 function NavigationLink( { icon } ) {
-	const { isRequestingSiteIcon, siteIconUrl } = useSelect( ( select ) => {
-		const { getEntityRecord, isResolving } = select( coreDataStore );
-		const siteData =
-			getEntityRecord( 'root', '__unstableBase', undefined ) || {};
+	const { isRequestingSiteIcon, postType, siteIconUrl } = useSelect(
```

Comment:
Nit: Since we're in a _template_ editor, I'd prefer it to be named `templateType`.

Reply:
Good point.

Btw selector names are a little confusing in the `edit-site` store.

Reply:
Yeah, most definitely 😆.

Reply:
Fixed in 933942f.